fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -128,6 +128,12 @@ public PlanFragment translatePlan(PhysicalPlan physicalPlan, PlanTranslatorConte
             rootFragment = exchangeToMergeFragment(rootFragment, context);
         }
         List<Expr> outputExprs = Lists.newArrayList();
+        if (physicalPlan instanceof PhysicalProject) {
+            PhysicalProject project = (PhysicalProject) physicalPlan;
+            if (isUnnecessaryProject(project)) {
+                physicalPlan = (PhysicalPlan) physicalPlan.child(0);
+            }
+        }
```

Comment:
after remove the root project node, we should set output exprs's order as them in the project to ensure the result is correct.